src/NewRelic.Telemetry/Logging/ITelemetryLogger.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+
+namespace NewRelic.Telemetry
+{
+#if INTERNALIZE_TELEMETRY_SDK
+    internal
+#else
+    public
+#endif
+    interface ITelemetryLogger
```

Comment:
Do we need this to be public for consumers of the Telemetry SDK?

Reply:
You're correct this doesn't need to be public either. However, making this internal requires additional changes to the DataSender because we can no longer reference that type in a protected .ctor or protected field.